settings.gradle

```diff
@@ -169,3 +169,13 @@ if (new File(rootDir, 'snappy-connectors/jdbc-stream-connector/build.gradle').ex
   project(':snappy-jdbc-connector_' + scalaBinaryVersion).projectDir =
       "$rootDir/snappy-connectors/jdbc-stream-connector" as File
 }
+
+if (new File(rootDir, 'snappy-connectors/gemfire-connector/build.gradle').exists()) {
+  include ':gemfire-connector'  
+  include ':gemfire-connector:connector'  
+  include ':gemfire-connector:gfeFunctions'  
+
+  project(':gemfire-connector').projectDir ="$rootDir/snappy-connectors/gemfire-connector" as File
```

Comment:
We have been appending scala binary version to the project names. Can use it here as well for consistency.

Reply:
Please add snappy-connectors in .gitignore

Reply:
Added scalaBinary version to the connector submodule generated jar.  The gfeFunctions.jar is totally a gemfire side artifact & depends only on java , so that is left unchanged.